wordle-web/app.vue

```diff
@@ -3,44 +3,79 @@
     <v-app>
       <v-app-bar color="primary" :elevation="2">
         <template v-slot:prepend>
-          <v-icon color="secondary"> mdi-book </v-icon>
+          <v-btn icon @click="router.push('/')">
+            <v-icon color="#FFFFFF">mdi-wizard-hat</v-icon> <!-- Wizard hat icon -->
+          </v-btn>
+          <v-btn @click="router.push('/')">
+            Welcome to LexiQuest
+          </v-btn>
         </template>
 
-        <v-app-bar-title>WordleApp</v-app-bar-title>
+        <v-spacer></v-spacer> <!-- This pushes the rest of the items to the right -->
 
-        <v-btn @click="router.push('/')">Home</v-btn>
-        <v-btn @click="router.push('/test')">Test</v-btn>
-        <v-btn icon="mdi-theme-light-dark" @click="toggleTheme" />
+        <v-btn icon="mdi-cog" @click="toggleSettingsDrawer" />
         <v-btn icon="mdi-help-circle" @click="showHelpDialog = true" />
+        <v-btn icon="mdi-menu" @click="toggleDrawer" />
+
         <HelpDialog v-model="showHelpDialog" />
       </v-app-bar>
+
+      <!-- Navigation Drawer for the About section -->
+      <v-navigation-drawer v-model="drawer" location="right">
+        <v-list>
+          <v-list-item @click="router.push('/about')" class="text-center">About</v-list-item>
+        </v-list>
+      </v-navigation-drawer>
```

Comment:
The about sidebar is drawn over the theme sidebar when both are clicked. It would be nice if the they closed when the other was clicked, but it's still very nice-looking so far.